pom.xml

```diff
@@ -68,7 +68,7 @@
       <plugin>
         <groupId>org.codehaus.gmaven</groupId>
         <artifactId>gmaven-plugin</artifactId>
-        <version>1.4</version>
+        <version>1.5-jenkins-3</version>
```

Comment:
Otherwise generated sources get put in a weird structure that confuses NetBeans at least.